plutus-core/plutus-core/src/PlutusCore/Crypto/Hash.hs

```diff
@@ -21,6 +24,14 @@ sha2_256 = digest (Proxy @SHA256)
 sha3_256 :: BS.ByteString -> BS.ByteString
 sha3_256 = digest (Proxy @SHA3_256)
 
+-- | Hash a [[BSL.ByteString]] using the Blake2B-224 hash function.
+blake2b_224 :: BS.ByteString -> BS.ByteString
+blake2b_224 = digest (Proxy @Blake2b_224)
+
 -- | Hash a [[BSL.ByteString]] using the Blake2B-256 hash function.
```

Comment:
Also it's not `BSL.ByteString` but rather `BS.ByteString`?

Reply:
I just changed it to `ByteString`. The signature should tell us everything we need to know about the type.